torchprime/metrics/xplane_pb2.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+
+# ruff: noqa
+
+# Generated by the protocol buffer compiler.  DO NOT EDIT!
```

Comment:
Is it possible to add a proto file? Not needed if it complicates the build.

Reply:
It probably requires a custom build step. Let me file an issue.